pandas/core/groupby/ops.py

```diff
@@ -424,8 +424,15 @@ def _get_cython_func_and_vals(
         return func, values
 
     def _cython_operation(
-        self, kind: str, values, how: str, axis: int, min_count: int = -1, **kwargs
-    ):
+        self, kind: str, values, how: str, axis, min_count: int = -1, **kwargs
+    ) -> Tuple[np.ndarray, Optional[List[str]]]:
```

Comment:
are you sure `Optional[List[str]]` is right?  I expected something like `List[Optional[Hashable]]`.

Reply:
oh, its only for ohlc.  never mind